ios/Podfile.lock

```diff
@@ -1343,4 +1343,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 05dc3e3e330cf28ff2c3e6b3517d100d067246b6
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.12.1
```

Comment:
Please upgrade your CocoaPods. Looks like you are on an older version.

Reply:
good catch. done